lib/matplotlib/axis.py

```diff
@@ -940,6 +940,12 @@ def set_tick_params(self, which='major', reset=False, **kwargs):
 
         For documentation of keyword arguments, see
         :meth:`matplotlib.axes.Axes.tick_params`.
+
+        See Also
+        --------
+        :meth:`.Axis.get_tick_params`
```

Comment:
```suggestion
        .Axis.get_tick_params
```
numpydoc does not need markup here (https://numpydoc.readthedocs.io/en/latest/format.html#see-also).